api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -19,7 +19,7 @@
 package v1alpha1
 
 import (
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
```

Comment:
```suggestion
	v1 "k8s.io/api/core/v1"
```

`make check` is failing without the import alias.